components/datatypes/internal-site-search.schema.json

```diff
@@ -0,0 +1,155 @@
+        },
+        "xdm:resultsPageNumber": {
+          "title": "Results Page Number",
+          "type": "integer",
+          "description": "For paginated search results, tracks the page of results the visitor is viewing."
+        },
+        "xdm:resultsPerPage": {
+          "title": "Results Per Page",
+          "type": "integer",
+          "description": "For paginated search results, tracks the number of search results displayed per page."
+        },
+        "xdm:instances": {
+          "title": "Instances",
+          "type": "integer",
+          "description": "Number of times an internal site search has occurred."
+        },
+        "xdm:locationInPage": {
+          "title": "Location in Page",
+          "type": "string",
+          "description": "When multiple search boxes exist on the page, this value should be used to identify the specific location the user used to search"
+        },
+        "xdm:nullInstances": {
+          "title": "Null Instances",
+          "type": "integer",
+          "description": "Number of times an internal site search has occurred that provided zero results."
+        },
+        "xdm:postalCode": {
+          "title": "Postal Code",
+          "type": "string",
+          "description": "Captures the zip code used for a search, if applicable"
+        },
+        "xdm:radiusDistance": {
+          "title": "Radius Distance",
+          "type": "integer",
+          "description": "Captures the selected distance of the search radius."
+        },
+        "xdm:radiusType": {
+          "title": "Radius Type",
+          "type": "string",
+          "enum": ["miles", "kilometers"],
+          "description": "Captures the selected distance type of the search radius in either miles or kilometers."
+        },
+        "xdm:term": {
+          "title": "Term",
+          "type": "string",
+          "description": "Tracks the internal site search term entered by the visitor."
+        },
+        "xdm:autocompleteClicked": {
+          "title": "Autocomplete Clicked",
+          "type": "boolean",
+          "description": "Indicates when a visitor uses a suggested/autocompleted search value to execute the search"
+        },
+        "xdm:autocompleteValue": {
+          "title": "Autocomplete Value",
+          "type": "string",
+          "description": "For autocomplete scenarios, users sometimes abandon their search and click a specific term from the drop down. This value is used to track the specific terms clicked"
+        },
+        "xdm:autocompleteTypedValue": {
+          "title": "Autocomplete Typed Value",
+          "type": "string",
+          "description": "For autocomplete scenarios, users sometimes abandon their search and click a specific term from the drop down. This value tracks what the user started to type in order to generate the specific set of suggested search terms"
+        },
+        "xdm:type": {
```

Comment:
This is to generic for a name. Should we rename this to "searchType" or something like that ?

Reply:
I've updated this in the next build.